examples/static_graph_optimizations/ptb/gentxt.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+"""Example to generate text from a recurrent neural network language model.
```

Comment:
Add `with static subgraph optimization` to make clear how this is different from the other one.